.gitignore

```diff
@@ -44,3 +40,14 @@ tmp
 **/test.log
 
 *.tsbuildinfo
+
+# Sewing Kit
+.sewing-kit
+build
+packages/*/*.esnext
+packages/*/*.node
+packages/*/*.mjs
+packages/*/*.d.ts
+packages/*/*.js
```

Comment:
The thing to watch out for here is that we sometimes nest `.eslintrc.js` or similar in subdirectories, and we do generally want to include those files.